homeassistant/components/kulersky/light.py

```diff
@@ -94,6 +67,11 @@ async def discover(*args):
     async_track_time_interval(hass, discover, DISCOVERY_INTERVAL)
 
 
+async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Cleanup the Kuler sky integration."""
+    hass.data.pop(DOMAIN, None)
```

Comment:
As for zerproc this needs to move to an `async_unload_entry` function in `__init__.py`.

Reply:
Oh yeah, I'll have that pushed up soon

Reply:
Pushed it up here https://github.com/home-assistant/core/pull/47572